File: Security_Tools/My Security tool/scripts/attack_site.pl
use LWP::UserAgent;
use HTTP::Request;

my $url = "http://localhost/pycyber1/test2.php";
my $user_agent = LWP::UserAgent->new;
my $response = $user_agent->get($url);

if($response->is_success){
    print "Server :".$response->decoded_content
}

File: Security_Tools/My Security tool/scripts/attack_site.py
# حظر الطلبات المشبوهة و منع استكشاف المسارات الحساسة

import requests 
import time

search = ["atr.html","login.php","main.txt","admin.php"]

url = "http://localhost/pycyber1/"

for x in search:
    full_url = url + x
    response = requests.get(full_url)
    if response.status_code == 200:
        print(f"[+]Found : {full_url} | code : {response.status_code}")
    else:
        print(f"[-]Not Found : {full_url} | code : {response.status_code}")
print(response.text)

